scripts/Engineering/gui/engineering_diffraction/tabs/calibration/model.py

```diff
@@ -0,0 +1,243 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2019 ISIS Rutherford Appleton Laboratory UKRI,
+#     NScD Oak Ridge National Laboratory, European Spallation Source
+#     & Institut Laue - Langevin
+# SPDX - License - Identifier: GPL - 3.0 +
+
+from __future__ import (absolute_import, division, print_function)
+
+from os import path, makedirs
+from ntpath import basename
+
+from mantid.api import AnalysisDataService as Ads
+from mantid.kernel import logger
+from mantid.simpleapi import Load, EnggVanadiumCorrections, EnggCalibrate, DeleteWorkspace, CloneWorkspace, \
+    CreateWorkspace, AppendSpectra
+from mantidqt.plotting.functions import plot
+from Engineering.EnggUtils import write_ENGINX_GSAS_iparam_file
+
+
+class CalibrationModel(object):
+    def __init__(self):
+        self.VANADIUM_INPUT_WORKSPACE_NAME = "engggui_vanadium_ws"
+        self.CURVES_WORKSPACE_NAME = "engggui_vanadium_curves"
+        self.INTEGRATED_WORKSPACE_NAME = "engggui_vanadium_integration"
+        self.out_files_root_dir = path.join(path.expanduser("~"), "Engineering_Mantid/")
```

Comment:
Can we extract out the `"Engineering_Mantid"` into a named variable at module scope? It improves readability if we have hard-coded strings all in once place with names. Also, if we are using `os.path` then we shouldn't ever need any trailing slashes.